01-JS-DOM-APIs/js/index.js

```diff
@@ -0,0 +1,90 @@
+function fade() {
+  setTimeout(function () {
+    document.getElementById("fade-in").classList.add("fade-in");
+  }, 2000);
+}
+
+document.getElementById("button-get-repositories").addEventListener("click", function() {
+  var keyword = document.getElementById("keyword-repositories").value;
+  if (keyword !== "") {
+    clearElements();
+    var config = {
+      "method": "GET",
+      "url": "https://api.github.com/search/repositories",
+      "asynchronous": true,
+      "params": { "q": keyword }
+    }
+    // We define what to do when the promise is resolved/fulfilled with the then() call,
+    // and the catch() method defines what to do if the promise is rejected.
+    performAJAXCall(config).then(
+      function(value) {
+        value = JSON.parse(value);
+        var node;
+        var repositoriesList = document.getElementById("repositories-list");
+        var textNode;
+
+        for(var key in value.items) {
+          node = document.createElement("li");
+          textNode = document.createTextNode(value.items[key].full_name);
+          node.appendChild(textNode);
+          repositoriesList.appendChild(node);
+        }
+      })
+    .catch(
+      // Log the rejection reason
+      function(reason) {
+        document.getElementById("fade-in").style.color = "red";
```

Comment:
You are using this `document.getElementById("fade-in")` twice. You should store this into a variable.